OpenRA.Mods.Common/Traits/BlockedByActor.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace OpenRA.Mods.Common.Traits
+{
+	// NOTE: Each subsequent category is a superset of the previous categories
+	// and categories are mutually exclusive.
+	public enum BlockedByActor
```

Comment:
I think we usually put stuff like this in `TraitsInterfaces.cs`.

Reply:
Done